lib/iris/util.py

```diff
@@ -1059,13 +1059,14 @@ def format_array(arr):
     ffunc = str
     max_line_len = 50
 
-    formatArray = np.core.arrayprint._formatArray
-    format_options = np.core.arrayprint._format_options
+    # Format the array with version 1.13 legacy behaviour
+    with np.printoptions(legacy="1.13"):
+        # Use this (private) routine for more control.
+        formatArray = np.core.arrayprint._formatArray
+        # N.B. the 'legacy' arg had different forms in different numpy versions
+        # -- fetch the required form from the internal options dict
+        legacy_key = np.core.arrayprint._format_options["legacy"]
 
-    options = np.get_printoptions()
```

Comment:
@pp-mo From what I recall (as I was last in here a few years ago)... it was important to call `np.get_printoptions` to seed the correct default values.

Not sure if that still the case, and still a safer thing to do

Reply:
> thanks to the context-manager
:+1: well that was my reading of it.
Anyway, it does seem to work.